components/Modal/Modal.jsx

```diff
@@ -194,24 +127,34 @@ class Modal extends Component {
   render() {
     const { children, onClose, closeButtonAriaLabel, theme, ...rest } =
       this.props;
+    const wrapperClass = classNames(styles.wrapper);
```

Comment:
```suggestion
    const { className = '', children, onClose, closeButtonAriaLabel, ...rest } =
      this.props;
    const wrapperClass = classNames(styles.wrapper, className);
```
It is very important to take the className here and pass it as the last argument of classNames. 

Possible problematic scenario: Currently the modal component has className="wrapper". I'm going to customize the Modal component and pass a className="my-customs" to it. When the component is rendered, its className will be equal to "my-customs" instead of "wrapper my-customs", as the {...rest} that is made below overwrites the component's default className. With the above tweak, we make sure this doesn't happen

Reply:
Done!